client/src/modules/cohort/components/CohortHeader.jsx

```diff
@@ -171,30 +187,32 @@ class CohortHeader extends PureComponent {
       updateBreadcrumbs,
       updateCohort
     } = this.props;
-    const { nameInputValue } = this.state;
+    const { errorMessageId, nameInputValue } = this.state;
     const nameToUpdate = _trim(nameInputValue);
     const { name: previousName } = details;
 
-    if (_trim(previousName) === nameToUpdate) {
-      this.setState({ isNameInputVisible: false });
+    if (!errorMessageId) {
+      if (_trim(previousName) === nameToUpdate) {
```

Comment:
We get `previousName` from props, from DB. Do we have to trim it? Cohort/list `name` should be saved to DB without trailing spaces so wonder if trimming `previousName` is necessary?

Reply:
It was here already :)